api/src/controllers/profile.ts

```diff
@@ -273,21 +272,19 @@ export const addContactToProfile = async (
 export const fetchProfileContacts = async (
   { params, user }: { params: any, user: AuthenticatedUser }, res: Response
 ): Promise<void> => {
-  const { ContactModel, ProfileModel } = dm;
+  const { ContactModel } = dm;
   const { profileId } = params;
 
   try {
-    const record = await ProfileModel.findById(Number(profileId));
+    const projectContacts = await ContactModel.findForProject(Number(profileId));
 
-    const notAuthorized = isNotAuthorized(record, user);
+    const isAuthorized = getAuthorization(profileId, user, undefined, projectContacts);
```

Comment:
Just nitpick on readability. Currently getAuthorization either throw an Error or return true. It's just a bit odd to "throw isAuthorized" when "not authorized"?